From 43c1b68788794216a2f9850e3946a598d0b3ce90 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Thu, 4 May 2017 15:17:51 +0300
Subject: [PATCH 1754/5242] ARM64: dts: imx8qm: add missing sai0 clk, rename
 asrc0 clk

commit  5c3d7f87dc60d67f56dee9e456ee1ea19f4c8aeb from
https://source.codeaurora.org/external/imx/linux-imx.git

sai0 node: add missing mclk0 clk.
asrc0 node: rename dma clk to spba.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
Reviwed-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi |    8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 7f03e26..7ddcb08 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1390,8 +1390,10 @@
 		interrupts = <GIC_SPI 314 IRQ_TYPE_LEVEL_HIGH>;
 		clocks = <&clk IMX8QM_AUD_SAI_0_IPG>,
 			<&clk IMX8QM_AUD_SAI_0_MCLK>,
-			<&clk IMX8QM_CLK_DUMMY>, <&clk IMX8QM_CLK_DUMMY>;
-		clock-names = "bus", "mclk1", "mclk2", "mclk3";
+			<&clk IMX8QM_CLK_DUMMY>,
+			<&clk IMX8QM_CLK_DUMMY>,
+			<&clk IMX8QM_CLK_DUMMY>;
+		clock-names = "bus", "mclk0", "mclk1", "mclk2", "mclk3";
 		dma-names = "rx", "tx";
 		dmas = <&edma2 12 0 1>, <&edma2 13 0 0>;
 		status = "disabled";
@@ -1427,7 +1429,7 @@
 			"asrck_4", "asrck_5", "asrck_6", "asrck_7",
 			"asrck_8", "asrck_9", "asrck_a", "asrck_b",
 			"asrck_c", "asrck_d", "asrck_e", "asrck_f",
-			"dma";
+			"spba";
 		dmas = <&edma2 0 0 0>, <&edma2 1 0 0>, <&edma2 2 0 0>,
 			<&edma2 3 0 1>, <&edma2 4 0 1>, <&edma2 5 0 1>;
 		dma-names = "rxa", "rxb", "rxc",
-- 
1.7.9.5

